src/mtev_http1.c

```diff
@@ -1052,6 +1082,17 @@ mtev_http1_session_req_consume_read(mtev_http1_session_ctx *ctx,
           ALLOC_BCHAIN(DEFAULT_BCHAINSIZE);
     }
     else if (tail->start + tail->size >= tail->allocd) {
+      /* expand req.*_input list
+       *
+       * (*) At least in the chunked case, we never get here.
+       * This code is executed only if the function is called with a full buffer before reading from
+       * the socket (below). However,
+       * - If we were holding a complete chunk in the buffer, we would have jumped this section (`goto` ~ L1038).
+       * - If we were holding an incomplete chunk, we expand the buffer to make it fit (`ALLOC_BCHAIN` ~ L1040).
```

Comment:
Again, these line number references are fragile.  Use a more resilient reference method... C[1] C[2] is possible.

Reply:
fixed